code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2611,3 +2611,41 @@
 		var/color
 		CONVERT_PH_TO_COLOR(exposed_atom.reagents.ph, color)
 		exposed_atom.add_atom_colour(color, WASHABLE_COLOUR_PRIORITY)
+
+/datum/reagent/ants
+	name = "Ants"
+	description = "A sample of a lost breed of Space Ants (formicidae bastardium tyrannus), they are well-known for ravaging the living shit out of pretty much anything."
+	reagent_state = SOLID
+	color = "#993333"
+	taste_mult = 1.3
+	taste_description = "<span class='warning'>ANTS OH GOD</span>"
+	metabolization_rate = 5 * REAGENTS_METABOLISM //1u per second
+	glass_name = "glass of ants"
+	glass_desc = "Bottoms up...?"
+	ph = 4.6 //Ants contain Formic Acid, meaning they would likely be very acidic themselves if you used them as a reagent
+	var/ant_damage = 0
+	var/amount_left = 0
+
+/datum/reagent/ants/on_mob_life(mob/living/carbon/victim)
+	victim.adjustBruteLoss(max(0.2, round((ant_damage * 0.01),0.1))) //Around 25 brute for 50 seconds
```

Comment:
same question about max/min here

Reply:
Oh and both this and the status effect can be active at  the same time right? so like 1 brute per second if someone makes ant gas or whatever?

Reply:
I never took that into consideration, I should lower the damages on both of them.